.gitignore

```diff
@@ -11,3 +11,4 @@ test_results/*
 *.core
 *.profraw
 .DS_Store
+/src/cpu-template-helper/src/utils/mock_kernel/kernel.bin
```

Comment:
Should this be *.bin? We should avoid checking in binaries.

Reply:
There are some other `*.bin` files. I can rename `kernel.bin` to `kernel.img` and put `*.img` in `.gitignore` instead.
```
$ find . -name *.bin
./src/cpu-template-helper/src/utils/mock_kernel/kernel.bin
./src/vmm/src/utilities/mock_resources/test_elf.bin
./src/vmm/src/utilities/mock_resources/test_noisy_elf.bin
./src/vmm/src/utilities/mock_resources/test_pe.bin
```

Reply:
Yeah we do have some already, but gitignore applies to any file not already part of the repo. This would help with accidentally checking binaries and make it a bit harder to check those in.

Reply:
that makes sense! added `*.bin` to `.gitignore`.